### PR TITLE
Follow-ups on ENT-8573 & ENT-8579

### PR DESCRIPTION
**Pull request by larsewi**
Follow-ups from https://github.com/cfengine/cfbs/pull/111

With the new Result namedtuple we can override the commit message during command execution.

Output with one update:
```
$ cfbs update
The default commit message is 'Update module 'promise-type-groups' to version 0.1.3' - edit it? [yes/y/NO/n] 
Committing using git:

[master 8a71ec6] Update module 'promise-type-groups' to version 0.1.3
 1 file changed, 2 insertions(+), 2 deletions(-)

```

Output with more than one update:
```
$ cfbs update
The default commit message is:

	Updated 2 modules

	 - Updated module 'promise-type-groups' to version 0.1.3
	 - Updated module 'promise-type-git' to version 0.1.1

Edit it? [yes/y/NO/n] 
Committing using git:

[master b2d3ea8] Updated 2 modules
 1 file changed, 4 insertions(+), 4 deletions(-)

```

### CODE REVIEW

File: cfbs/git_magic.py [hunk 1]
```diff
@@ -4,12 +4,16 @@
...
 """
 
+from collections import namedtuple
 from cfbs.prompts import YES_NO_CHOICES, prompt_user
 from cfbs.cfbs_config import CFBSConfig, CFBSReturnWithoutCommit
 from cfbs.git import git_commit, git_discard_changes_in_file, CFBSGitError
 from functools import partial
 
 
+Result = namedtuple("Result", ["rc", "commit", "msg_body"])
```
Comment (olehermanse) [inline cfbs/git_magic.py:14]: I don't like that you can only set the body, but not the title (first line) of the commit message in the result. I'd want to be able to create different commit messages (including title) depending on what happened in the function.

**Review by olehermanse — CHANGES_REQUESTED** [review]
Looks good in general, but I think we should enable setting the whole commit message in the result.

File: cfbs/git_magic.py (hunk 1 above)
Comment (larsewi) [reply cfbs/git_magic.py]: Now you override the entire commit message if 
```Python
msg is not None
```

**Review by olehermanse — APPROVED** [review]
LGTM

**Review by vpodzime — COMMENTED** [review]
Very nice! :clap:
